xcode/Subconscious/Shared/Components/Settings/GatewayURLSettingsView.swift

```diff
@@ -52,7 +34,10 @@ struct GatewayProvisionLabel: View {
                             .repeatForever(autoreverses: false)
                             .speed(0.4), value: spin)
                         .onAppear() {
-                            self.spin = true
+                            // Avoids unwanted animation of position after layout
```

Comment:
Tried my best adding fixed height frame's everywhere but nothing seems to help. I think it's the _position_ not the _size_ that's changing because the icon appears at exactly the middle of the display and then animates to where it should be.

Regardless, `.task{}` also works and is cleaner.